benchmarks/maxtext_xpk_runner.py

```diff
@@ -361,25 +375,42 @@ def generate_xpk_workload_cmd(
       ' https://raw.githubusercontent.com/GoogleCloudPlatform/ai-on-gke/9ff340f07f70be0130454f9e7238551587242b75/scripts/network-setup/v6e-network-optimization.yaml'
   )
 
+  # pathways-related flags
+  pathways_specific_flags = ''
+  docker_image_flag = f'--base-docker-image="{BASE_DOCKER_IMAGE}"'
+  if use_pathways:
+    pathways_specific_flags = (
+        ' --use-pathways'
+        f' --additional_pw_proxy_args={reformat_xla_flags_for_xpk(model.xla_flags)}'
```

Comment:
I think you need hyphens instead of underscores here in "--additional_pw_proxy_args"